deps/Makefile

```
@@ -170,6 +174,30 @@ zstd/zstd/lib/libzstd.a:
+		-DCMAKE_INSTALL_PREFIX=$(PROTOBUF_INSTALL_DIR) \
+		-DCMAKE_POSITION_INDEPENDENT_CODE=ON \
+		-Dprotobuf_BUILD_TESTS=OFF \
+		-Dprotobuf_BUILD_CONFORMANCE=OFF \
+		-Dprotobuf_BUILD_EXAMPLES=OFF \
+		-Dprotobuf_BUILD_PROTOC_BINARIES=OFF \
+		-Dprotobuf_BUILD_LIBPROTOC=OFF \
+		-Dprotobuf_BUILD_SHARED_LIBS=OFF \
+		-Dprotobuf_WITH_ZLIB=OFF
+	cd protobuf/protobuf-$(PROTOBUF_VERSION) && CC=${CC} CXX=${CXX} ${MAKE} -C build install
+
+protobuf: $(PROTOBUF_LIB)
```

checkmake (0.3.2)

[warning] Target "protobuf" should be declared PHONY.

(phonydeclared)